### ISSUE TITLE
Add mirror to VAM scene

### ISSUE THREAD

**Issue by chris-b-pacon** [pinosante/VAM-Evolutionary-Character-Creation#24]
Already suggested in forum -- good idea.

**Comment by pinosante** [comment]
Yeah, I think I have such a version laying around somewhere actually :).

**Comment by chris-b-pacon** [comment]
It is a must have, and of course I am too stupid to do it for myself. :)

Seriously, not everyone has a "creator" license, I assume. This would be nice for the "teasers".
